<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image matcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="imageScrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1600</width>
      <height>600</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>118</width>
       <height>78</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="imagesSelectionBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>620</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="runMatchingBtn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>700</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run image matching</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="neighborhoodSizeTextEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>810</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="neighborhoodConsistencyThresholdTextEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>870</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="neigborhoodSizeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>820</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Neighborhood size</string>
    </property>
   </widget>
   <widget class="QLabel" name="neighborhoodConsistencyLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>880</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Neighborhood consistency threshold</string>
    </property>
   </widget>
   <widget class="QComboBox" name="matchingMethodComboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>700</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Neighborhood consistency</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RANSAC Affine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RANSAC Perspective</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="neighborhoodConsistencyOptionsLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>760</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Neighborhood consistency method options</string>
    </property>
   </widget>
   <widget class="QLabel" name="ransacOptionsLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>760</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RANSAC options</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorThresholdLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>820</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Error threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="itersNumLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>870</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations number</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="errorThresholdTextEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>820</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="itersNumTextEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>870</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="distanceHeuristicCheckBox">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>760</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Points pairs distance heuristic (RANSAC)</string>
    </property>
   </widget>
   <widget class="QLabel" name="distanceLowerBoundLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>820</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance lower bound</string>
    </property>
   </widget>
   <widget class="QLabel" name="distanceUpperBoundLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>870</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance upper bound</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="distanceLowerBoundTextEdit">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>820</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="distanceUpperBoundTextEdit">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>870</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="itersNumEstimationHeuristicCheckBox">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>760</y>
      <width>301</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Iterations number estimation (RANSAC)</string>
    </property>
   </widget>
   <widget class="QLabel" name="pProbabilityLabel">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>820</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Probability that model will be good enough after estimated iters num</string>
    </property>
   </widget>
   <widget class="QLabel" name="wProbabilityLabel">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>870</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Probability that selected pair is not noise</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pProbabilityTextEdit">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>820</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="wProbabilityTextEdit">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>870</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="timeElapsedLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>620</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Elapsed time [s]:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="elapsedTimeText">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>620</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mutualNeighborhoodPointsPairsLabel">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>620</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mutual neighborhood points pairs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageACharacteristicPointsNumLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>620</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image A characteristic points num:</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageBCharacteristicPointsNumLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>620</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Image B characteristic points num:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="imageACharacteristicPointsNumText">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>620</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="imageBCharacteristicPointsNumText">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>620</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="mutualNeighborhoodPairsNumText">
    <property name="geometry">
     <rect>
      <x>1370</x>
      <y>620</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="finalMatchingPointsNumLabel">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>680</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Final matching points num:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="finalMatchingPointsNumText">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>680</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ransacPerformedItersLabel">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>700</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RANSAC performed iterations num:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ransacPerformedItersText">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>700</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
